assets/css/styles.css

```diff
@@ -411,6 +411,15 @@ h1.title {
 /*===================================*
 0. FOOTER
 *===================================*/
+.img-fluid{
+  max-width:40vw;
+ 
+  
+}
```

Comment:
Please remove this class, and instead add `max-width: 40vw;` to the `.footer-logo` class